index.d.ts

```diff
@@ -3,7 +3,7 @@ import * as ReactNative from "react-native";
 
 declare module "react-native-linear-gradient" {
 
-    export interface LinearGradientProps extends ReactNative.ViewProperties {
+    export type LinearGradientProps = Exclude<ReactNative.ViewProps, 'start' | 'end'> & {
```

Comment:
- changed `ViewProperties` to `ViewProps`. `ViewProperties` is deprecated and actually just points to `ViewProps` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/legacy-properties.d.ts#L107
- excluded `start` & `end` from `ViewProps`